build-presets/MakePresetTree.py

```diff
@@ -103,12 +102,11 @@
             for ilr in inverter_loading_ratio:
                 if ilr not in preset_tree[cell_technology][package_type][system_type]:
                     preset_tree[cell_technology][package_type][system_type][ilr] = {}
-                for solar_resource_dict in weather_data:
-
-                    json_model.SolarResource.solar_resource_data = solar_resource_dict
-                    lat = json_model.SolarResource.solar_resource_data['lat']
-                    lon = json_model.SolarResource.solar_resource_data['lon']
-                    location = locations[(lat, lon)]  # string name of location
+                for f in weather_files:
+                    json_model.SolarResource.solar_resource_data = tools.SAM_CSV_to_solar_data(f)
+                    f = f.replace(weather_folder + '/', '') # remove folder name from file name
+                    id = f.split('_')[0] # get the id from the beginning of the string
```

Comment:
`id` is another python built in we should avoid reassigning, maybe `location_id` or something?